src/DacpacTool/PackageBuilder.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System;
 using System.IO;
+using System.IO.Packaging;
```

Comment:
I don't understand this... the commit message says "respond to breaking changes", so is this change related to some upstream dependency?

Reply:
I think the DacFX package no longer includes a copy of the Packaging API

Reply:
Yeah, I think that's what happend as well. After upgrading the DacFX package the  `Microsoft.Data.Tools.Schema.Sql.Packaging` no longer existed. So I figured I'd just replace it with the built-in `System.IO.Packaging` API to see if that works. It turned out to be basically a drop-in replacement. I'm assuming that when they originally built DacFX that the Packaging API didn't exist in .NET yet. But now it does so they decided to replace their own implementation with the one from the framework.

Reply:
I had the exact same experience when updating my .dacpac reverse engineering library.